x/gov/keeper/tally.go

```diff
@@ -97,7 +97,7 @@ func (keeper Keeper) Tally(ctx sdk.Context, proposal types.Proposal) (passes boo
 
 	// If there is not enough quorum of votes, the proposal fails
 	percentVoting := totalVotingPower.Quo(keeper.sk.TotalBondedTokens(ctx).ToDec())
-	if percentVoting.LT(tallyParams.Quorum) {
+	if hasReachedQuorum(percentVoting, tallyParams.Quorum) {
```

Comment:
What I was trying to suggest is this
```suggestion
        quorum := tallyParams.GetQuorum(proposal.IsExpedited)
	if percentVoting.LT(quorum) {
```

Similar to:
```go
threshold := tallyParams.GetThreshold(proposal.IsExpedited)
```

Reply:
i see that makes sense. Just changed